examples/deno-deploy/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@examples/deno-deploy",
+  "version": "10.3.0",
+  "private": true,
+  "type": "module",
+  "scripts": {
+    "dev": "deno run --allow-net=:8000 --allow-env ./src/index.ts",
+    "test-dev": "start-server-and-test 'pnpm dev' http://127.0.0.1:8000 'tsx src/client.ts'"
+  },
+  "dependencies": {
+    "@trpc/client": "^10.3.0",
+    "@trpc/server": "^10.3.0",
+    "zod": "^3.0.0"
+  },
+  "devDependencies": {
+    "@types/node": "^18.7.20",
+    "@types/node-fetch": "^2.5.11",
+    "node-fetch": "^2.6.1",
+    "start-server-and-test": "^1.12.0",
+    "tsx": "^3.9.0",
+    "typescript": "^4.8.3"
+  },
+  "publishConfig": {
+    "access": "restricted"
+  }
+}
```

Comment:
Do deno have `package.json`s? 🤔

Reply:
Absolutely, we can remove it but then turborepo won't be able to run the `dev` and `test-dev` pipeline in this example.

Reply:
If we want to test deno we should probably add that so it works differently

Reply:
The `test-dev` scripts - from both this new deno & vercel examples **and** from CF workers - aren't working right now. I'll take a look at `src/client.ts` for each of these examples when this is merged

Reply:
But can we not just remove this file for now?